mem0/memory/main.py

```diff
@@ -109,6 +110,13 @@ def add(
                 {"role": "user", "content": prompt},
             ]
         )
+
+        # Check if the extracted memories where already saved, return if they are found.
+        if self.db.search_history(extracted_memories):
```

Comment:
can you please check this "if" statement? Check whether it is getting True or False as desired for a few test cases.

Reply:
Yes, I have tested it throughly, it works fine.